lib/datasource/pod/index.ts

```diff
@@ -98,14 +101,17 @@ const githubRegex = regEx(
   /^https:\/\/github\.com\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
 );
 
+const githubEnterpriseRegex = regEx(
+  /(?<hostURL>(^https:\/\/github\..*\.com))\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
+);
+
 async function getReleasesFromGithub(
   lookupName: string,
-  registryUrl: string,
+  match: RegExpExecArray,
```

Comment:
Thanks, @ranolf for the suggestion:

```suggestion
  opts: { hostURL: string; account: string; repo: string; useShard: boolean }
```